en/reference/services-search.html

```diff
@@ -328,6 +329,44 @@ <h2 id="renderer">renderer</h2>
           bundle="the name in &lt;artifactId&gt; in pom.xml" /&gt;
 </pre>
 
+<h2 id="significance">significance</h2>
+<p>
+The significance element can include multiple models. Their order determines the model precedence for a given language, with the last element having the highest. The models' document frequency is used to set a token's significance. To enable the use of these models, the schema needs to have a rank-profile with the <em>significance</em> element and the  <em>use-model</em> set to <em>true</em>.
```

Comment:
This section should link to both `significance.html` and `reference/schema-reference.html#significance`.